python/cuml/cuml/svm/linear.pyx

```diff
@@ -643,22 +644,92 @@ class LinearSVM(Base, metaclass=WithReexportedParams):
         self.classes_ = None
         self.probScale_ = None
 
+    @classmethod
+    def _get_param_names(cls):
+        return super()._get_param_names() + [
+            "penalty",
+            "loss",
+            "fit_intercept",
+            "penalized_intercept",
+            "probability",
+            "max_iter",
+            "linesearch_max_iter",
+            "lbfgs_memory",
+            "C",
+            "grad_tol",
+            "change_tol",
+            "epsilon",
+            "tol"
+        ]
+
+    @classmethod
+    def _params_from_cpu(cls, model):
```

Comment:
This doesn't seem to be complete. A quick look through shows at least one parameter we'll want to raise `UnsupportedOnGPU` for (`intecept_scaling != 1.0`). It also looks like `multi_class` might be fully ignored (cuml docs and afaict implementation only support `multi_class="ovr"`, but never seem to check it afaict? We probably also want to fallback on `multi_class="crammer_singer"` (for LinearSVC, not the base class).

Reply:
Addressed in 122eb382a07215dadc1a107c521baa57d835d344 and 11efb9a651bbfb1c9cda550acd70ce12b6808d36 . I think you actually caught both cases, but I might have missed some.